portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase

```diff
@@ -849,13 +849,17 @@ definition {
 		}
 
 		task ("Open URL and LogIn") {
-			Open.openNoError(locator1 = ${workspaceURL});
+			Open(locator1 = ${workspaceURL});
```

Comment:
Can we keep the Open.openNoError?

Reply:
Yeah, we can. I ran it both ways locally and both passed, but this may be necessary for something in CI, right? Better not to change something that is working there 🤣